### PR TITLE
#131356211 Coveralls Code Coverage

### PR DESCRIPTION
**Pull request by andela-foladeji**
What does this PR do?

Integrates coveralls code coverage and adds the badge to readme

Description of Task to be completed?

The coveralls code coverage service is integrated to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product

How should this be manually tested?

By building the app with travis and see what the badge shows

Any background context you want to provide?

NA

What are the relevant pivotal tracker stories?

#131356211 Integrate Coveralls code coverage service

Screenshots (if appropriate)

NA

Questions:

NA

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ var require = require(),
     io = require('socket.io');
 require('dotenv').config();
 
+require('dotenv').config();
```
Comment (q-ode) [inline server.js]: You might want to take a second look at this.
I think we'll be getting duplicates into the repo.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/hyuga-cfh/badge.svg)](https://coveralls.io/github/andela/hyuga-cfh)
+
 Cards for Humanity - [http://cfh.io](http://cfh.io)
 ===========
 Cards for Humanity is a fast-paced online version of the popular card game, Cards Against Humanity, that gives you the opportunity to donate to children in need - all while remaining as despicable and awkward as you naturally are.
 
+[![Build Status](https://travis-ci.org/andela/hyuga-cfh.svg?branch=master)](https://travis-ci.org/andela/hyuga-cfh)
```
Comment (q-ode) [inline README.md]: Same as previous comment, likely duplicates.

File: server.js (hunk 1 above)
Comment (andela-foladeji) [inline server.js]: This was probably because I merged staging with my branch yesterday before pushing. I will do a rollback

File: README.md (hunk 2 above)
Comment (andela-foladeji) [inline README.md]: Will fix this too

File: .gitignore [hunk 3]
```diff
@@ -3,5 +3,5 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
-.env
+*.env
```
Comment (andela-foladeji) [inline .gitignore]: The * was to use a wildcard to ensure that files ending with .env don't get tracked by git. However, on second thought, I think there can only be 1 .env file so the .env is not wrong either

**Comment by andela-foladeji** [comment]
Needed some refactoring

**Pull request closed without merging**